tests/src/Kernel/fixtures/markup_rendering_patterns/timeline.yml

```diff
@@ -0,0 +1,183 @@
+timeline_items_hide:
+  render:
+    '#type': pattern
+    '#id': timeline
+    '#fields':
+      settings:
+        hide_from: 7
+        hide_to: 0
+      toggle_collapsed: "Show %d more items"
+      toggle_expanded: "Hide %d items"
+      items:
+        - label: "13 September 2017"
+          title: "Item 1"
+          content:
+            '#markup': '<a href="/example" class="ecl-link">Economic and Monetary Union package of proposals</a>'
+        - label: "28-29 September 2017"
+          title: "Item 2"
+          content: "Lorem Ipsum 2"
+        - content: "Lorem Ipsum 3"
+        - content: "Lorem Ipsum 4"
+        - content: "Lorem Ipsum 5"
+        - label: "6 February 2018"
+          title: "Item 6"
+          content: "Lorem Ipsum 6"
+  assertions:
+    count:
+      'ol.bcl-timeline': 1
+      'li h5': 6
+      'li a': 1
+      'button': 0
+      '.collapse': 0
+    equals:
+      '.bcl-timeline li:nth-child(1) h5': "13 September 2017"
+      '.bcl-timeline li:nth-child(1) h6': "Item 1"
+      '.bcl-timeline li:nth-child(1) div': '<a href="/example" class="ecl-link">Economic and Monetary Union package of proposals</a>'
+      '.bcl-timeline li:nth-child(2) h5': "28-29 September 2017"
+      '.bcl-timeline li:nth-child(2) h6': "Item 2"
+      '.bcl-timeline li:nth-child(2) div': 'Lorem Ipsum 2'
+      '.bcl-timeline li:nth-child(3) h5': ''
+      '.bcl-timeline li:nth-child(3) div': 'Lorem Ipsum 3'
+      '.bcl-timeline li:nth-child(4) h5': ''
+      '.bcl-timeline li:nth-child(4) div': 'Lorem Ipsum 4'
+      '.bcl-timeline li:nth-child(5) h5': ''
+      '.bcl-timeline li:nth-child(5) div': 'Lorem Ipsum 5'
+      '.bcl-timeline li:nth-child(6) h5': '6 February 2018'
+      '.bcl-timeline li:nth-child(6) h6': 'Item 6'
+      '.bcl-timeline li:nth-child(6) div': 'Lorem Ipsum 6'
```

Comment:
'.label-expanded': "Display %d more"
 '.label-collapsed': "Stash %d items"
replace with %d actually is, if the button doesn't show in this case use it on one of the others, the points is to test also the toggle button overrides not just the default values.